code/datums/components/hat_stabilizer.dm

```diff
@@ -146,3 +151,17 @@
 	if (ismob(apparel.loc))
 		var/mob/wearer = apparel.loc
 		wearer.update_clothing(wearer.get_slot_by_item(apparel))
+
+// maybe we don't need the item context proc but instead the hand one? since we don't need to check held_item
```

Comment:
This comment doesn't make any sense in this context, it says we don't check held item but we... do?

Reply:
Sloppiness by me. I ported over this code from my rotation component refactor and forgot to clean it up.

Reply:
Classic, I do it all the time